pcf/quasiparticle/cross_cloud/cross_cloud_storage/cross_cloud_storage.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2018 Capital One Services, LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pcf.core.quasiparticle import Quasiparticle
+from pcf.particle.gcp.storage.bucket import Bucket
+from pcf.particle.aws.s3.s3_bucket import S3Bucket
+
+
+class CrossCloudStorage(Quasiparticle):
+    """
+    This quasiparticle helps manage data across AWS S3 and GCP Storage
+    """
+    flavor = "cross_cloud_storage"
+
+    def __init__(self, particle_definition):
+
+        particle_definition['particles'] = {
+            "flavor":"s3_bucket",
+            "aws_resource":{
+                "Bucket":"pcf-testing"}
```

Comment:
this is hard coded. this is should be replaced by the pcf_name or "name" field in quasiparticle config